crates/forge-runner/src/cairo_runner/sierra_casm_runner.rs

```diff
@@ -164,25 +174,68 @@ impl SierraCasmRunner {
         })
     }
 
-    /// Handling the main return value to create a `RunResultValue`.
-    #[allow(clippy::cast_sign_loss)]
-    fn handle_main_return_value(
+    /// Extract inner type if `ty` is a panic wrapper
+    fn maybe_inner_type(&self, ty: &GenericTypeId, func: &Function) -> Option<ConcreteTypeId> {
```

Comment:
```suggestion
    fn maybe_inner_type(&self, ty: &GenericTypeId, func: &Function) -> Option<ConcreteTypeId> {
```
Please don't use maybe prefix, it is already encoded in type info. Name doesn't suggest what this function does. What is `inner_type`?

Reply:
If return type is a panic wrapper, it wraps some actual return type. It was already named `inner_type` in cairo repo code.

Reply:
Renamed